Tools/wasm/mypy.ini

```diff
@@ -0,0 +1,15 @@
+[mypy]
+files = Tools/wasm
+pretty = True
+show_traceback = True
+
+# Make sure the wasm can be run using Python 3.8:
+python_version = 3.8
+
+# Be strict...
+strict = True
```

Comment:
Will this mean **all** code in this directory must be fully typed?

Reply:
We can ignore any code that is not typed:
1. Via adding global `disallow_incomplete_defs = false`
2. Via per-file ignores
3. Via inline `# type: ignore`

For now, yes, all code must be typed. Do you want to add `disallow_incomplete_defs = false`?

Reply:
> Do you want to add `disallow_incomplete_defs = false`?

Yes please.